### PR TITLE
Add rockmongo Dockerfile

### PR DESCRIPTION
**Pull request by derekwaynecarr**

### CODE REVIEW

File: centos-rockmongo/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# VERSION      1
```
Comment (smarterclayton) [inline centos-rockmongo/Dockerfile]: Nuke this?

File: centos-rockmongo/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+FROM centos
```
Comment (smarterclayton) [inline centos-rockmongo/Dockerfile:1]: Add maintainer

File: centos-rockmongo/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+# update
+RUN yum -y update
```
Comment (smarterclayton) [inline centos-rockmongo/Dockerfile]: do this in a one liner with all yum tasks

File: centos-rockmongo/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+# install required
+RUN yum install -y httpd php php-devel wget php-pear unzip gcc-c++ 
+
+#install mongo drivers
+RUN pecl install mongo
+RUN cd /etc && echo "extension=mongo.so" >> /etc/php.d/mongo.ini
```
Comment (smarterclayton) [inline centos-rockmongo/Dockerfile]: Do this in a single line

File: centos-rockmongo/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+# install RockMongo
+RUN cd /root && wget -O rockmongo-1.1.5.zip http://rockmongo.com/downloads/go?id=12 && unzip rockmongo-1.1.5.zip -d /var/www/ && rm -R /var/www/html && mv /var/www/rockmongo/ /var/www/html
+
+# increase php upload size 
+RUN sed -i 's/upload_max_filesize = 2M/upload_max_filesize = 10M/g' /etc/php.ini
```
Comment (smarterclayton) [inline centos-rockmongo/Dockerfile]: oneline this

File: centos-rockmongo/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+RUN sed -i 's/post_max_size = 2M/post_max_size = 10M/g' /etc/php.ini
+
+RUN echo '<?php phpInfo(); ?>' > /var/www/html/info.php
+
+CMD ["/usr/sbin/httpd", "-D", "FOREGROUND"]
```
Comment (smarterclayton) [inline centos-rockmongo/Dockerfile:23]: expose your port

**Comment by derekwaynecarr** [comment]
completed updates requested

File: centos-rockmongo/Dockerfile [hunk 7]
```diff
@@ -0,0 +1,22 @@
+FROM centos
+
+MAINTAINER Derek Carr <decarr@redhat.com>
+
+# update, install required, clean
+RUN yum -y update; yum install -y httpd php php-devel wget php-pear unzip gcc-c++; yum clean all
```
Comment (smarterclayton) [inline centos-rockmongo/Dockerfile]: You need to && - the default shell execution isn't -e so you wouldn't get warned of failures in between

**Comment by derekwaynecarr** [comment]
made last requested update.

**Pull request merged**